src/ipc/ipc4/dai.c

```diff
@@ -88,6 +88,7 @@ int ipc_dai_data_config(struct comp_dev *dev)
 		copier_cfg = dd->dai_spec_config;
 		/* set dma burst elems to slot number */
 		dd->config.burst_elems = copier_cfg->base.audio_fmt.channels_count;
+		dev->ipc_config.frame_fmt = SOF_IPC_FRAME_S32_LE;
```

Comment:
I think we need a comment to state the text from the commit message to the code.

Reply:
sure